install-monorepo.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+# Print help and exit.
+function usage {
+	cat <<-EOH
+		usage: $0 
+
+		Installs all the required tooling for the Jetpack Monorepo. 
+	EOH
+	exit 1
+}
+
+if [[ $1 ]]; then
+	usage
+fi
+
+# Check if we're on a Mac or Linux, bail if we're not.
+OS="$(uname)"
+if [[ "$OS" == "Linux" ]]; then
+	ON_LINUX=1
+	sudo apt install build-essential
```

Comment:
I added those checks, but stopped short of installing them ourselves in the script figuring there must be some point we have to cut off checking a never ending tree of dependencies, unless you think otherwise.

Reply:
I suggested `git` and `curl` since those were the two things I had to install in a fresh docker image when testing this. But if you don't want to, that's ok too.